com.is_slick.rmac.rmac.metainfo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <id>com.is_slick.rmac.rmac</id>
+  
+  <name>rmac</name>
+  <summary>Assembler for 680x0 processors (and others found in Atari machines)</summary>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+  
+  <description>
+    <p>
+      rmac is a macro assembler targetting the 680x0, 6502, Jaguar JRISCs (codenamed Tom and Jerry), Jaguar Object Processor, and the DSP 56001 CPUs. It can output to standard aout object file format, ELF, or directly to Atari TOS PRG format or absolute binary.
+    </p>
+  </description>
+  
+  <icon type="stock">com.is_slick.rmac.rmac</icon>
+  
```

Comment:
Missing a bunch of metadata, validate and fill them in https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#validation